x/liquidity/spec/01_concepts.md

```diff
@@ -1,60 +1,85 @@
+Liquidity providers deposit the two kinds of coins into the liquidity pool and then share the accumulated swap fee with respect to their pool share. Pool share is represented as possession of pool coins.
 
 ### Coin Swap
 
-Users can request coin swap to a liquidity pool on an AMM style exchange without interacting with constantly changing orderbooks. The requested coin swap is executed with a swap price calculated from given swap price function, the current other swap requests and the current liquidity pool coin reserve status. Swap orders are executed only when execution swap price is equal or better than submitted order price of the swap order.
+You can request coin swap to a liquidity pool on an AMM style exchange without using a classic order book mechanism.
+
+1. The requested coin swap is executed with a swap price that is calculated from the given swap price function.
+
+2. The current other coin swap requests and the current liquidity pool coin reserve status.
+
+3. Swap orders are executed only when the execution swap price is equal to or greater than the submitted order price of the swap order.
 
 ### Price Discovery
 
-Coin swap prices in liquidity pools are determined by the current liquidity pool coin reserves and current requested swap amount. Arbitrageurs constantly buy or sell coins in liquidity pools to gain instant profit which results in real-time price discovery of liquidity pools.
+Coin swap prices in liquidity pools are determined by the current liquidity pool coin reserves and current requested swap amount. Arbitrageurs constantly buy or sell coins in liquidity pools to gain instant profit that results in real-time price discovery of liquidity pools.
 
 ### Escrow Process
 
-For swap order, deposit and withdrawal messages, the module escrow necessary amount of coins from users' balance to ensure action commitments from message senders.
+For swap orders and deposit and withdrawal messages, the module withdraws the escrow amount of coins from your balance.
 
 ### Swap Fees
 
-Coin swap requestors pay swap fees to liquidity pools, which are accumulated in the liquidity pools so that ultimately the pool coin owners will accumulate profit from them.
+Coin swap requestors pay swap fees to liquidity pools. Swap fees are accumulated in the liquidity pools so that the pool coin owners accumulate profit.
 
 ### Batches and Swap Executions
 
-Coin swaps are executed for every batch, which is composed of one or more consecutive blocks. The size of each batch can be decided by governance parameters and the algorithm in the liquidity module.
+Coin swaps are executed for every batch. A batch is composed of one or more consecutive blocks. The size of each batch can be decided by governance parameters and the algorithm in the liquidity module.
 
 ### Pool Identification
```

Comment:
what is this pool identification category? let's add an intro sentence to why I care about this information

Reply:
This `Pool Identification` section is needed for those who may be curious about how `PoolName`, `PoolReserveAcc`, and `PoolCoinDenom` is created. Maybe we can have [appendix.md](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/SPEC-SPEC.md#common-layout). What do you think @dongsam ?

Reply:
Yes. Let's use smaller PRs to tackle this.